models/classes/user/LtiUserHelper.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 (original work) Open Assessment Technologies SA ;
+ */
+
+namespace oat\taoLti\models\classes\user;
+
+
+use oat\generis\model\GenerisRdf;
+use oat\oatbox\service\ConfigurableService;
+use oat\oatbox\service\exception\InvalidService;
+
+/**
+ * Class LtiUserHelper
+ * @package oat\taoLti\models\classes\user
+ */
+class LtiUserHelper extends ConfigurableService
```

Comment:
The name of the `service` is `Helper`, it's misleading

Reply:
tbh I don't think that we need this helper at all, why can't we just use `LtiUserService`?